packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -30,13 +30,6 @@ import { SubscriptionTier } from '../db/types/hbarLimiter/subscriptionTier';
 import { IDetailedHbarSpendingPlan } from '../db/types/hbarLimiter/hbarSpendingPlan';
 import { ConfigService } from '@hashgraph/json-rpc-config-service/dist/services';
 
-/**
```

Comment:
Q: why the removal of the class comments?

Reply:
Accidentally removed when the `HBAR_SPENDING_CONFIG` property initial reading was moved to the `loadSpendingPlansConfig` method.  Restored.